doc/api/worker_threads.md

```diff
@@ -1008,7 +1008,10 @@ changes:
   * `execArgv` {string\[]} List of node CLI options passed to the worker.
     V8 options (such as `--max-old-space-size`) and options that affect the
     process (such as `--title`) are not supported. If set, this is provided
-    as [`process.execArgv`][] inside the worker. By default, options are
+    as [`process.execArgv`][] inside the worker. `--experimental-loader`
+    and `--import` with `module.register()` calls to contribute to the hooks
```

Comment:
Is `--import` which has no effect or is it the `module.register()`?

Reply:
strictly from code opint of view it is the `module.register()` but in terms of expectations with worker `execArgv` (that this refers to) it is the combination of the two that does not have effect.

Reply:
In that case I find it a bit misplaced here as I would assume only very few users of workers actually use customization hooks. One might argue that setting `process.title` in an `--import` script does also not work so text here could get long over time.

I would prefer to improve the documentation at `module.register` (already done here) and in the `Customization Hooks` section. The `Customization Hooks` section has no example for workers.
Also the sample `Communication with module customization hooks` does not mention how to setup this from a worker (if it is possible at all).
In the `Hooks` section there I read `The hooks thread may be terminated by the main thread at any time` - which doesn't seem correct.

Also the behavior in case loader thread hangs or dies is not described.

Usually one would expect that workers are isolated but via customization hooks all of them are bound to the loader thread and via this one to main thread.

But not sure if it is your intention to get that far.

Reply:
I think let’s just cut the addition to `worker_threads.md` and focus on `module.md`?